CHANGES

```diff
@@ -1,3 +1,9 @@
+Version 0.4.1
```

Comment:
for next time: 

for a whole new feature i might have gone for a larger bump, e.g. `0.5.0`.